build.sbt

```diff
@@ -8,14 +8,15 @@ val commonSettings = List(
   scalaVersion := scala212,
   crossScalaVersions := Seq(scala213, scala212),
   organization := "org.gnieh",
+  headerLicense := Some(HeaderLicense.ALv2("2020", "Lucas Satabin")),
```

Comment:
2021 by now? The files in this PR have a mix 🤨

Reply:
Well, this branch started in 2020, and I didn't upgrade ^^